lib/screens/settings/settings_screen.dart

```diff
@@ -56,6 +52,32 @@ class SettingsScreen extends StatelessWidget {
     );
   }
 
+  void _showConfirmLogoutDialog(BuildContext theContext){
```

Comment:
`theContext` -> `context`
I missed it previously, so sorry for this👀

Reply:
That is not concerned with this PR. So lets leave this for now.

Reply:
Actually the next function  which is _showChangedPassowrdDialog also has theContext as BuildContext so I used the same convention.

Reply:
Can you change everything to ```context```

Reply:
also your ```_showConfirmLogoutDialog``` has ```context``` too

Reply:
okay no problem.